public/index.html

```diff
@@ -40,6 +40,18 @@ <h2>About Cohort</h2>
           <p>Cohort is maintained by <a href="https://twitter.com/jakemoves">Jacob Niedzwiecki</a>. It consists of a Node+Express server and client packages for iOS (ObjC & Swift), web (JS), and Unity (C#). Sound, video, and AR content can be cued on clients from QLab or the Cohort web platform. Features currently under development include push notifications, geolocation, and VR/360 video.</p>
         </div>
       </div>
+        
```

Comment:
I'm being picky mostly since we're testing process; can you please use two spaces as indentation and indent this new section accordingly?

Reply:
Yeah, absolutely. I've been in Python for too long.